core/ops_builtin.rs

```diff
@@ -67,30 +66,16 @@ pub fn op_void_sync() {}
 pub async fn op_void_async() {}
 
 /// Remove a resource from the resource table.
-#[op]
-pub fn op_close(
-  state: &mut OpState,
-  rid: Option<ResourceId>,
-) -> Result<(), Error> {
-  // TODO(@AaronO): drop Option after improving type-strictness balance in
-  // serde_v8
-  let rid = rid.ok_or_else(|| type_error("missing or invalid `rid`"))?;
-  state.resource_table.close(rid)?;
-  Ok(())
+#[op(fast)]
+pub fn op_close(state: &mut OpState, rid: u32) -> bool {
```

Comment:
The op macro does handle `ResourceId` so there's no reason to revert to `u32` here.